### PR TITLE
docs: ✏️ Provide documentation on how to use db-radio

### PR DESCRIPTION
**Pull request by mfranzke**

### CODE REVIEW

File: packages/components/src/components/radio/docs/Angular.md [hunk 1]
```diff
@@ -1,8 +1,75 @@
...
+@NgModule({
+  …
+  imports: [ FormsModule, …]
+})
+```
+
+```html
+<!-- form.component.html-->
+<form>
+	<DBRadio ngDefaultControl [(ngModel)]="radio">Label</DBRadio>
```
Comment (nmerget) [inline packages/components/src/components/radio/docs/Angular.md:50]: Have you tested this? What you get with this is just "on". As a developer I don't have any clue which radio is selected. In elements we used a fromControl to get around this issue: https://github.com/db-ui/elements/blob/main/showcase/angular-showcase/src/app/components/form/form.component.html#L23

File: packages/components/src/components/radio/docs/React.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
 ## React
 
-TODO
+Load SCSS globally somewhere in your app:
```
Comment (nmerget) [inline packages/components/src/components/radio/docs/React.md:3]: Again, load it once in an `index.scss` inside `main.tsx/jsx`

File: packages/components/src/components/radio/docs/React.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+@use "@db-ui/foundations/build/scss/variables.global" as *;
+@use "@db-ui/components/build/styles/db-ui-42-rollup" as *;
+@use "@db-ui/foundations/build/scss/color-classes" as *;
+```
+
+Import component:
+
+```typescript
+import { DBRadio } from "@db-ui/react-components";
+
```
Comment (nmerget) [inline packages/components/src/components/radio/docs/React.md:15]: We would use a state here with a map and multiple radio buttons.
This wouldn't be a good example. May I edit it, or shall we do it together?

File: packages/components/src/components/radio/docs/Vue.md [hunk 4]
```diff
@@ -1,14 +1,36 @@
 ## Vue
 
-To get DBRadio work with `v-model` you have to use v-model argument syntax:
+Load SCSS globally somewhere in your app:
```
Comment (nmerget) [inline packages/components/src/components/radio/docs/Vue.md:3]: Load it in `index.scss/css` and import it in your `main.ts/js`

File: packages/components/src/components/radio/docs/Vue.md [hunk 5]
```diff
@@ -1,14 +1,36 @@
...
-<DBRadio label="Textlabel" v-model:checked="vModelTest"></DBRadio>
+```scss
+@use "@db-ui/foundations/build/scss/variables.global" as *;
+@use "@db-ui/components/build/styles/db-ui-42-rollup" as *;
+@use "@db-ui/foundations/build/scss/color-classes" as *;
+```
+
+Use component:
+
+```html
```
Comment (nmerget) [inline packages/components/src/components/radio/docs/Vue.md:13]: Isn't there a `vue` file ending?

File: packages/components/src/components/radio/docs/Vue.md [hunk 6]
```diff
@@ -1,14 +1,36 @@
...
+<script>
+	import { DBRadio } from "@db-ui/v-components";
+</script>
+
+<template>
+	<DBRadio value="value">Label</DBRadio>
```
Comment (nmerget) [inline packages/components/src/components/radio/docs/Vue.md:19]: This is a nonworking example as well. But I need to test it by myself to have a solution

File: packages/components/src/components/radio/docs/Angular.md [hunk 7]
```diff
@@ -1,8 +1,75 @@
 ## Angular
 
-Third party controls require a ControlValueAccessor to function with angular forms. Adding an `ngDefaultControl` attribute will allow them to use that directive.
+Load SCSS globally somewhere in your app:
```
[the remark this answers is not in the document]
Comment (mfranzke) [reply packages/components/src/components/radio/docs/Angular.md:3]: could you please propose a code change directly, as I'm not totally sure what you mean?

File: packages/components/src/components/radio/docs/React.md (hunk 3 above)
Comment (mfranzke) [reply packages/components/src/components/radio/docs/React.md:13]: please proceed.

**Review by nmerget — APPROVED** [review]

**Pull request merged**